engine/api/warning/warning_project.go

```diff
@@ -53,6 +66,63 @@ func computeWithProjectEvent(db gorp.SqlExecutor, e sdk.Event) error {
 	return nil
 }
 
+func manageProjectAddPermission(db gorp.SqlExecutor, key string, gp sdk.GroupPermission) error {
+	if err := removeWarning(db, MissingProjectPermissionEnv, gp.Group.Name); err != nil {
+		return sdk.WrapError(err, "manageProjectAddPermission> Unable to remove warning")
```

Comment:
I think in the log message you should indicate kind of warning, better for debug. Same for the next one